python/compiler/aiecc/cl_arguments.py

```diff
@@ -142,6 +142,41 @@ def parse_args(args=None):
             default=False,
             action='store_true',
             help='Show progress visualization')
+    parser.add_argument('--aie-generate-ipu',
+            dest="ipu",
+            default=False,
+            action='store_const', const=True,
+            help='Generate ipu instruction stream')
+    parser.add_argument('--aie-only-generate-ipu',
+            dest="only_ipu",
+            default=False,
+            action='store_const', const=True,
+            help='Generate ipu instruction stream only')
+    parser.add_argument('--ipu-insts-name',
+            dest="insts_name",
+            default="ipu_insts.txt",
+            help='Output instructions filename for IPU target')
+    parser.add_argument('--aie-generate-cdo',
+            dest="cdo",
+            default=False,
+            action='store_const', const=True,
+            help='Generate libxaie v2 for CDO')
+    parser.add_argument('--xclbin-name',
+            dest="xclbin_name",
+            default="final.xclbin",
+            help='Output xclbin filename for CDO/XCLBIN target')
+    parser.add_argument('--xclbin-kernel-name',
+            dest="kernel_name",
+            default="PP_FD_PRE",
+            help='Kernel name in xclbin file')
+    parser.add_argument('--xclbin-instance-name',
+            dest="instance_name",
+            default="MLIRAIE",
+            help='Instance name in xclbin metadata')
+    parser.add_argument('--xclbin-kernel-id',
+            dest="kernel_id",
+            default="0x201",
+            help='Kernel id in xclbin file')
```

Comment:
FYI there's a change here vs. what we had before. I added `xclbin-` prefix to these three arguments.  @jgmelber

Reply:
The default kernel name should be MLIR_AIE here

Reply:
And kernel ID is 0x901

Reply:
Thanks. And `MLIRAIEV1` instance name? I think I rebased wrong commits in this PR and I'm still picking through the wreckage.

Reply:
Yes that’s right